backend/review/management/commands/precompute_pcr_views.py

```diff
@@ -0,0 +1,101 @@
+import logging
+
+from django.core.cache import cache
+from django.core.management.base import BaseCommand
+from django.db import transaction
+from django.http import Http404
+from tqdm import tqdm
+
+from courses.models import Topic
+from review.models import CachedReviewResponse
+from review.views import manual_course_reviews
+
+
+def precompute_pcr_views(verbose=False, is_new_data=False):
+    if verbose:
+        print("Now precomputing PCR reviews.")
+
+    CachedReviewResponse.objects.all().update(expired=True)
+    responses = CachedReviewResponse.objects.all()
```

Comment:
nit: I got a little confused bc this is called responses. maybe cached_responses is a better name?